kconfigs/9pfs.conf

```diff
@@ -0,0 +1,4 @@
+CONFIG_NET_9P=y
```

Comment:
I don't see any issue with always having that enabled., so I'd prefer putting those configs in the already existing conf
https://github.com/OP-TEE/build/blob/master/kconfigs/qemu.conf